test/ensembl.test.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const _ = require('lodash');
+
+// Module where function we want to test (ApiConnection is passed as a parameter to uploadFile)
+const ensembl = require('../src/ensembl');
+const { ApiConnection } = require('../src/graphkb');
+// Same hostname as mock data required
+const HOSTNAME = 'bcgsc.ca';
+const conn = new ApiConnection(`http://${HOSTNAME}:8080/api`);
+
+
+// Module we want to mock (corresponding file needed in '../src/__mock__' folder)
+jest.mock('../src/util');
```

Comment:
Do you need to mock the entire util module? could you just mock the request function?

Reply:
ex, https://stackoverflow.com/questions/59312671/mock-only-one-function-from-module-but-leave-rest-with-original-functionality

and then in the test file

```js
jest.mock('../src/util.js', () => ({
    ...jest.requireActual('../src/util.js'),
    request: jest.fn(),
}));


describe('uploadFile', () => {

});


describe('fetchAndLoadById', () => {

});
```

Reply:
see newer comment and example PR instead: https://github.com/bcgsc/pori_graphkb_loader/pull/75